src/views/ModelPlan/TaskList/Payment/Recover/index.tsx

```diff
@@ -0,0 +1,416 @@
+                            onChange={() => {
+                              setFieldValue(
+                                'anticipateReconcilingPaymentsRetrospectively',
+                                true
+                              );
+                            }}
+                          />
+                          <Field
+                            as={Radio}
+                            id="payment-anticipate-reconciling-payment-retro-No"
+                            name="payment-anticipate-reconciling-payment-retro"
+                            label={h('no')}
+                            value="NO"
+                            checked={
+                              values.anticipateReconcilingPaymentsRetrospectively ===
+                              false
+                            }
+                            onChange={() => {
+                              setFieldValue(
+                                'anticipateReconcilingPaymentsRetrospectively',
+                                false
+                              );
+                            }}
+                          />
+                        </Fieldset>
+                        <AddNote
+                          id="payment-anticipate-reconciling-payment-retro-note"
+                          field="anticipateReconcilingPaymentsRetrospectivelyNote"
+                        />
+                      </FieldGroup>
+
+                      <FieldGroup
+                        scrollElement="payment-payment-start-date"
+                        error={!!flatErrors.paymentStartDate}
+                        className="margin-top-4"
+                      >
+                        <Label
+                          htmlFor="payment-payment-start-date"
+                          className="maxw-none"
+                        >
+                          {t('paymentStartDate')}
+                        </Label>
+                        <p className="text-normal margin-bottom-1 margin-top-0">
+                          {t('paymentStartDateSubcopy')}
+                        </p>
+                        <div className="usa-hint" id="appointment-date-hint">
+                          {h('datePlaceholder')}
+                        </div>
+                        <FieldErrorMsg>
+                          {flatErrors.paymentStartDate}
+                        </FieldErrorMsg>
+                        <Field
+                          as={DatePicker}
+                          error={+!!flatErrors.paymentStartDate}
+                          className="width-card-lg"
+                          id="payment-payment-start-date"
+                          maxLength={50}
+                          name="paymentStartDate"
+                          defaultValue={values.paymentStartDate}
+                          onBlur={(e: any) =>
+                            handleOnBlur(e.target.value, 'paymentStartDate')
+                          }
+                        />
```

Comment:
This datepicker form element should utilize the warning for dates in the past.  I completed this as a part of my work for OpsEvalAndLearning, specifically the page for IDDOC.  The component is called `DatePickerWarning` and requires a piece of state management.  You can reference the implementation for IDDOC.

Reply:
The translation for the validDate is not working when an error is thrown
![Screen Shot 2022-07-20 at 3 41 56 PM](https://user-images.githubusercontent.com/95709965/180067940-f687fed0-17ff-4fe4-9062-7027510a6642.png)